### DISCUSSION TITLE
Taxonomy filter for Loop Carousel

### DISCUSSION THREAD

**Discussion by samuelcerezo** [elementor/elementor#24759] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I want to to filter the elements of Loop Carousel widget. For example, that visitor can filter to only views posts from a specific category.

### Describe the solution you'd like.

The existing taxonomy filter widget should be able to work with Loop Carousel too, not only with Loop Grid.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by strony-love** [comment]
I would like to see that functionality in elementor pro as well.

It is especially useful for mobile where a scrolling loop carousel is better way to display instead of nonscrolling loop grid I.e. you can swipe left and right.
